lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/IncludeExcludeWidget.py

```diff
@@ -0,0 +1,77 @@
+from ..common.list import StringList
+from ..common.resizable import Resizable
+from ..common.kebabMenu import KebabMenu
+from .ExpressionRulesMenu import ExpressionMenu
+
+try:
+    from PySide6.QtWidgets import QFrame, QWidget, QHBoxLayout, QVBoxLayout# type: ignore
+except ImportError:
+    from PySide2.QtWidgets import QFrame, QWidget, QHBoxLayout, QVBoxLayout # type: ignore
+
+from pxr import Usd
+
+class IncludeExcludeWidget(QWidget):
+    def __init__(self, collection: Usd.CollectionAPI = None, parent: QWidget = None):
+        super(IncludeExcludeWidget, self).__init__(parent)
+        self._collection = collection
+
+        shouldIncludeAll = False
+        includes = []
+        excludes = []
+
+        if self._collection is not None:
+            includeRootAttribute = self._collection.GetIncludeRootAttr()
+            if includeRootAttribute.IsAuthored():
+                shouldIncludeAll = self._collection.GetIncludeRootAttr().Get()
+
+            for p in self._collection.GetIncludesRel().GetTargets():
+                includes.append(p.pathString)
+            for p in self._collection.GetExcludesRel().GetTargets():
+                excludes.append(p.pathString)
+
+        includeExcludeLayout = QVBoxLayout(self)
+        includeExcludeLayout.setContentsMargins(0,0,0,0)
+
+        self._expressionMenu = ExpressionMenu(self._collection, self)
+        menuButton = KebabMenu(self._expressionMenu, self)
+
+        separator = QFrame()
+        separator.setFrameShape(QFrame.VLine)
+
+        headerWidget = QWidget(self)
+        headerLayout = QHBoxLayout(headerWidget)
+        headerLayout.addStretch(1)
+        # Will need a separator for future designs
+        #headerLayout.addWidget(separator)
+        headerLayout.addWidget(menuButton)
+        includeExcludeLayout.addWidget(headerWidget)
+
+        self._include = StringList(includes, "Include", "Include all", self)
```

Comment:
I would have like the widget class to have names that reflect they are UI element. StringList and Resizable are very generic, at first I did not understand what was going on.

Reply:
Those were there on the initial version of this widget that was moved to maya. So I kept them with the same name for ease of updating